client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/style.scss

```
@@ -54,4 +54,9 @@
 		margin: auto;
 		padding: 8px 0;
 	}
+
+	.migration-message__actions {
+		display: flex;
+		gap: 1em;
+	}
```

From a visual perspective, I think we should add in more space above the button group, and expand the space between the two buttons.
```suggestion
	.migration-message__actions {
		display: flex;
		gap: 2em;
		margin-top: 3em; /* could also work with padding-top */
	}
```